R/pkg/R/DataFrame.R

```diff
@@ -3715,3 +3715,34 @@ setMethod("rollup",
             sgd <- callJMethod(x@sdf, "rollup", jcol)
             groupedData(sgd)
           })
+
+#' hint
+#'
+#' Specifies execution plan hint on the current SparkDataFrame.
```

Comment:
the R programming model is a bit different - I think it is better to point out the original SparkDataFrame is not actually changed - instead say `.... hint and return a new SparkDataFrame` is better